lib/releaselib.sh

```diff
@@ -116,10 +117,12 @@ release::set_build_version () {
   local main_job="${all_jobs[0]}"
 
   # Loop through the remainder, excluding anything specified by --exclude_suites
-  for ((i=1;i<=${#all_jobs[*]};i++)); do
-    re="\\b${all_jobs[$i]}\\b"
-    [[ ${exclude_suites[*]} =~ $re ]] || secondary_jobs+=(${all_jobs[$i]})
+  for ((i=1;i<${#all_jobs[*]};i++)); do
+    [[ -n $exclude_patterns && ${all_jobs[$i]} =~ $exclude_patterns ]] \
+        || secondary_jobs+=(${all_jobs[$i]})
   done
+  
+  logecho "After excluding $exclude_patterns, secondary_jobs are ${secondary_jobs[*]}"
```

Comment:
This list is generated as part of the normal output.  We don't need to display it here.